api/src/util/newrelic/events.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+from types import TracebackType
+from typing import Any, Dict, Optional, Tuple, Union
```

Comment:
While we're borrowing code, let's cleanup the typing, should be able to trim out dict, optional, union for the simpler forms (`dict`, `x | None`, `x | y` respectively)

Reply:
Got this too.